eng/jacoco-test-coverage/pom.xml

```diff
@@ -360,7 +360,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-spring-data-cosmos-core</artifactId>
-      <version>3.0.0-beta.2</version> <!-- {x-version-update;com.azure:azure-spring-data-cosmos-core;current} -->
+      <version>3.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-spring-data-cosmos-core;current} -->
```

Comment:
Same here, we don't need these version increments.

Reply:
Fixed

Reply:
Fixed